commcare_export/commcare_hq_client.py

```diff
@@ -52,7 +52,10 @@ def _get_auth(self, username, password, mode):
     @property
     def session(self):
         if self.__session == None:
-            self.__session = requests.Session(headers={'User-Agent': 'commcare-export/%s' % commcare_export.__version__})
+            self.__session = requests.Session()
+            self.__session.headers.update({
+                'User-Agent': 'commcare-export/%s' % commcare_export.__version__
+            })
         return self.__session
```

Comment:
huh, does `requests.Session()` specify some defaults that the old version overwrote?